projects/elife.yaml

```diff
@@ -654,6 +656,7 @@ journal:
                     fallbacks:
                         4xx: "4xx.html"
                         5xx: "5xx.html"
+                ip-blacklist: true
```

Comment:
since ip addresses lists are managed outside of builder/Terraform, this is the place to put a link to the Fastly panel where they are listed; not meant to be a long term solution